honeybadger-io/honeybadger-js
# chore(react-native): Minor updates to example projects

## Status
READY

## Description
Recording updates I've made that make it easier to use the test projects. 

Also removing an unused plugin (react-native-dotenv) that was causing an error. I'd uninstalled the dependency but forgot to remove the plugin. 


## Todos

## Steps to Test or Reproduce
Run the example projects -- you should see the ability to configure your revision, and no unexpected errors.
